### ISSUE TITLE
JAWS does not announce the navigation region in dialog

### ISSUE THREAD

**Issue by kabsas** [FreedomScientific/standards-support#758]
### Summary
JAWS does not announce the navigation region in dialog. This happens in Chrome and Edge with the virtual cursor off. However, with the VPC on, the navigation region is not announced when the dialog opens if the focused element is in the region, it is only announced when you navigate away from the region then back to it. In Firefox, the navigation region is not announced with VPC off and VPC on.

Steps to reproduce:

1. Launch JAWS
2. Go to https://codepen.io/yououhan123/pen/vYvmwGq
3. Turn off virtual cursor (Ctrl+Z)
4. Navigate to Open dialog button and activate the button

### Expected result
When the dialog opens the navigation region around the Home and About links is announced.

### Actual result
When the dialog opens the navigation region is not announced.

#### JAWS version and build number
2023.2302.15

#### Operating System and version
Windows 10 Enterprise 19045.3324

#### Browser and version:
Chrome 116
Edge 116
Firefox 117

**Comment by stevefaulkner** [comment]
further discussion needed, and further test creation needed

**Comment by kabsas** [comment]
@stevefaulkner - A navigation landmark should be announced when you enter the region. The problem is the navigation region is not announced when it is in a dialog as described in the summary. This is not an issue when the landmark is not in a dialog. We have come across several bugs in dialogs, such as https://github.com/FreedomScientific/standards-support/issues/706, that are quite problematic for our applications that contain complex content in dialogs.

**Comment by stevefaulkner** [comment]
@BrettLewisVispero thinks it should be working, will review

**Comment by BrettLewisVispero** [comment]
Imported into ADO as bug 79583
